NohBoard/Forms/Properties/MouseElementPropertiesForm.cs

```diff
@@ -142,6 +142,25 @@ private void btnAddBoundary_Click(object sender, EventArgs e)
             this.DefinitionChanged?.Invoke(this.currentDefinition);
         }
 
+        /// <summary>
+        /// Handles updating a boundary, sets the new boundaries and invokes the changed event.
+        /// </summary>
+        private void btnUpdateBoundary_Click(object sender, EventArgs e)
+        {
+            if (this.lstBoundaries.SelectedItem == null) return;
+
+            var updateIndex = this.lstBoundaries.SelectedIndex;
```

Comment:
`btnAddBoundary_Click` has a check whether the same boundary doesn't already exist. Perhaps a similar check should be in here.

Reply:
Good call. I'll add it when I have time tomorrow evening.